src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RoundUp.txt

```diff
@@ -230,3 +230,8 @@ Error({Kind:ErrorKind.InvalidArgument})
 >> RoundUp("true","false")
 Error(Table({Kind:ErrorKind.InvalidArgument},{Kind:ErrorKind.InvalidArgument}))
 
+>> ForAll(Sequence(16,-1,-1), RoundUp( Float(123.12345678919), Value ))
+Table({Value:130},{Value:200},{Value:1000},{Value:10000},{Value:100000},{Value:1000000},{Value:10000000},{Value:100000000},{Value:1000000000},{Value:10000000000},{Value:100000000000},{Value:1000000000000},{Value:10000000000000},{Value:100000000000000},{Value:1000000000000000},{Value:10000000000000000})
+
+>> ForAll(Sequence(16,-1,-1), RoundUp( Float(-123.12345678919), Value ))
```

Comment:
>ForAll(Sequence(16,-1,-1), RoundUp( Float(-123.12345678919), Value )) [](http://example.com/codeflow?start=3&length=69)

The RoundUp function does have a tabular overload, so this can be written in a more concise way as

    `RoundUp( -Sqrt(123.45*123.45), Sequence( 16,-1,-1 ) )`